### ISSUE TITLE
Different padding used in finetuning and pretraining

### ISSUE THREAD

**Issue by chenzhekl** [facebookresearch/sapiens#235]
Thanks for this great project!

I noticed that no padding was used for pre-training, while padding=2 was set for all fine-tuning tasks. IIUC, the image size used in fine-tuning (1024x768) is perfectly divisible by the patch size (16). Why did you choose to intentionally pad the input to make it not divisible by the patch size?

https://github.com/facebookresearch/sapiens/blob/08dce797f7b40f5b41388f518cac85535c3f5d13/seg/configs/sapiens_normal/normal_general/sapiens_1b_normal_general-1024x768.py#L67
